### DISCUSSION TITLE
Multi-line taskbar

### DISCUSSION THREAD

**Discussion by SZLMCL** [valinet/ExplorerPatcher#680] [Ideas]
Hello, 
In Windows 10 we can create multi-line taskbar, but this is not possible in Windows 11.
Can you please develop this function? Thanks!

**Comment by Anixx** [comment]
In what sense the current functionality is not satisfactory for you?

![изображение](https://user-images.githubusercontent.com/1271736/149676729-19b6fdfd-7ae0-4077-b8b9-55825a34bb56.png)

**Reply by SZLMCL** [reply]
Ohh sorry, this is perfect, my taskbar was locked, therefore I not see this function. Thank you!
